src/api/app/views/webui/users/tokens/edit.html.haml

```diff
@@ -23,6 +23,12 @@
               .form-group
                 = f.label(:type, 'Operation:')
                 = @token.class.token_name.capitalize
+              .form-group
+                = f.label(:name, 'Name:')
+                %span.d-none#original-token-name
```

Comment:
What is the reason for having this? The token name is displayed in the input below when the page is loaded, so I suggest removing this.

Reply:
That's used after clicking in "Regenerate Token". Instead of having this:

![Screenshot from 2021-11-02 11-49-28](https://user-images.githubusercontent.com/24919/139833433-a1fb2f9a-6bfa-4be3-afc5-f2a499e85e21.png)

we would have something similar to this:

![Screenshot from 2021-11-02 11-54-33](https://user-images.githubusercontent.com/24919/139833808-f20f588d-77b7-4820-9de3-ddbcde3e07aa.png)

Reply:
Okay, this isn't obvious at all... but there's not much we can do for this now in this PR. Thanks for the explanation.